android-plugin/src/main/java/io/ecocode/xml/Xml.java

```diff
@@ -20,10 +20,10 @@
 package io.ecocode.xml;
 
 public interface Xml { //NOSONAR - we use the interface for constant without inheriting it
-    String REPOSITORY_NAME = "ecoCode";
-    String PROFILE_NAME = "ecoCode";
+    String REPOSITORY_NAME = "ecoCode (Android)";
+    String PROFILE_NAME = "ecoCode (Android)";
     String KEY = "xml";
-    String REPOSITORY_KEY = "ecoCode-xml";
-    String XML_RESOURCE_PATH = "io/ecocode/xml/rules";
+    String REPOSITORY_KEY = "ecocode-android-xml";
+    String XML_RESOURCE_PATH = "io/ecocode/rules/xml";
     String PROFILE_PATH = "io/ecocode/xml/rules/ecocode_xml_profile.json";
```

Comment:
why do we keep old path ?

Reply:
I don't understand your question.
Indeed, I change the path of rules to the new one: `io/ecocode/rules/xml`

Reply:
ok for path in `XML_RESOURCE_PATH`: it's the new path.
but what about path in variable `PROFILE_PATH` (line not modified, thus old path for me) : why do we have a new path like `io/ecocode/rules/xml` (variable `XML_RESOURCE_PATH `)  and another like `io/ecocode/xml/rules/...` (variable `PROFILE_PATH `)
why don't we make homogenous all these paths with new pattern `.../rules/xml/...` ?

Reply:
I think this (implementation-related) profile should not be the same as the specification path.
Thus, the profile must share the same path as the implementation and not the same specification path.
I will change my path based on this explanation.

Reply:
Personally, I would have modified all paths to the same pattern to be simpliest and not to ask ourselves in the future my question above :p 
So, the most important for me is that all repositories must have the same rules to name their paths.